.github/workflows/ci-build.yaml

```diff
@@ -252,7 +252,7 @@ jobs:
         run: |
           set -x
           cd "$GITHUB_WORKSPACE/applicationsets"
-          kubectl apply -f manifests/crds/argoproj.io_applicationsets.yaml
+          kubectl create -f manifests/crds/argoproj.io_applicationsets.yaml
```

Comment:
This will probably fail if the build image already has this CRD installed. Is it guaranteed that we are always getting a fresh image?

Reply:
Looks to me as if it always spins up a fresh k3s before running the tests.